--- libosmscout/maps/build.sh.0002~	2016-08-23 01:24:43.687391374 +0200
+++ libosmscout/maps/build.sh	2016-08-23 01:25:21.123016536 +0200
@@ -58,6 +58,6 @@ echo " $outputFile" | tee -a $outputFile
 echo "Options:" | tee -a $outputFile
 echo " $options" | tee -a $outputFile
 echo "Call:" | tee -a $outputFile
-echo " ../Import/src/Import $options --typefile ../stylesheets/map.ost --destinationDirectory $targetDirectory $@" | tee -a $outputFile
+echo " Import $options --typefile ../stylesheets/map.ost --destinationDirectory $targetDirectory $@" | tee -a $outputFile
 
-../Import/src/Import $options --typefile ../stylesheets/map.ost --destinationDirectory "$targetDirectory" "$@" 2>&1 | tee -a $outputFile
+Import $options --typefile ../stylesheets/map.ost --destinationDirectory "$targetDirectory" "$@" 2>&1 | tee -a $outputFile
