--- libosmscout/libosmscout/configure.ac.omv~	2014-12-23 15:09:20.774437283 +0100
+++ libosmscout/libosmscout/configure.ac	2014-12-23 15:09:35.924552307 +0100
@@ -47,7 +47,7 @@ AC_ARG_ENABLE([marisa-support],
 AS_IF([test "$enable_cpp0x_support" != "no"],
       [AX_CHECK_COMPILE_FLAG([-std=c++0x],
                              [CPP0XFLAGS="-std=c++0x"
-                              CPPFLAGS="$CPPFLAGS -std=c++0x"
+                              CXXFLAGS="$CXXFLAGS -std=c++0x"
                               AC_SUBST(CPP0XFLAGS)])])
 
 AC_TYPE_SIZE_T
