--- libosmscout/stylesheets/map.ost.omv~	2015-01-04 23:36:22.375210967 +0100
+++ libosmscout/stylesheets/map.ost	2015-01-04 23:42:15.882872609 +0100
@@ -478,6 +478,21 @@ TYPES
       ADDRESS
 
   //
+  // Contour Lines
+  //
+  TYPE contour_minor
+    = WAY ("contour_ext" == "elevation_minor")
+      PATH
+
+  TYPE contour
+    = WAY ("contour_ext" == "elevation_medium")
+      PATH
+
+  TYPE contour_major
+    = WAY ("contour_ext" == "elevation_major")
+      PATH
+
+  //
   // Landuses
   //
 
--- libosmscout/stylesheets/standard.oss.omv~	2015-01-04 23:42:23.269690191 +0100
+++ libosmscout/stylesheets/standard.oss	2015-01-04 23:45:17.979025958 +0100
@@ -270,6 +270,9 @@ CONST
   COLOR landColor             = #fcfbe7;
   COLOR unknownColor          = #c4dff6;*/
 
+  // Contour lines
+  COLOR contourColor               = #d39f4d;
+
   // Landuse, water
   COLOR allotmentsColor            = #c8b084;
   COLOR beachColor                 = #eecc55;
@@ -485,6 +488,24 @@ SYMBOL speed_camera
    [TYPE natural_peak] NODE.ICON { symbol: natural_peak; }
  }
 
+ [MAG state-] {
+   [TYPE contour_major] {
+     WAY { color: @contourColor; displayWidth: 0.5mm; }
+   }
+ }
+
+ [MAG county-] {
+   [TYPE contour] {
+     WAY { color: @contourColor; displayWidth: 0.3mm; }
+   }
+ }
+
+ [MAG city-] {
+   [TYPE contour_minor] {
+     WAY { color: @contourColor; displayWidth: 0.1mm; }
+   }
+ }
+
  // -------------------------------------------------------
  //
  // waterway_*
